.github/workflows/ci.yml

```diff
@@ -0,0 +1,186 @@
+name: CI
+
+on:
+    push:
+        branches: [master]
+    pull_request:
+        branches: [master]
+
+jobs:
+    Lint-Shellcheck:
```

Comment:
Any reason why we call the jobs capitalizing? Is there a convention or something? Also note "Lint-Shellcheck" vs "Lint-black"

Reply:
I don't think there is a convention, just maintaining the sentence case. will do `Lint-shellcheck` 👍

Reply:
IMHO these jobs are called in such a way because they appear in the workflow job list like here in the left: https://github.com/reanahub/reana/actions/runs/329122827

Also, it seems there are no strict naming convention for `yaml` files: https://stackoverflow.com/questions/22771226/what-is-the-canonical-yaml-naming-style so I guess we need to select one and use it across the projects

Reply:
Yeah, we can just group similar kinds of steps together and follow sentence case for the naming. wdyt?

Reply:
I would vote for using all lowercase, for simplicity.

P.S. BTW what about making PRs against `maint-0.7`? We shall need to provide support for this branch, so GA there would be handy for longer time.  This seems to be a good example of "make PR to the lowest reasonable `maint-X.Y` branch where the fix may be applicable".)

Reply:
okay, going with lowercase 👍 .

I am still curious why GH actions don't trigger for my PR 🤔. @audrium Do you see anything that I might be missing?

Reply:
I can change the base of the PR to `maint-0.7`

Reply:
I think it can only be triggered once the workflow file is already in the master branch. I hope so :)